server/app.js

```diff
@@ -547,6 +548,12 @@ app.get('/api/advancedSearchSuggest', craftService.getAutoSuggestions);
 app.get('/api/tour/:slug', tourService.getTour);
 app.get('/api/eye-spy/:id', tourService.getTour);
 
+/** Endpoint for retrieving asset information from the NetX DAMS */
+app.get('/api/objects/:id/assets', async (request, response) => {
+  const result = await damsService.getAssetByObjectId(request.params.id);
+  response.json(result);
+});
+
```

Comment:
This endpoint will fetch renditions on Artwork Object Panel Detail load, and return them from NetX